ttnn/cpp/ttnn/operations/eltwise/binary/device/binary_composite_op.cpp

```diff
@@ -373,7 +373,7 @@ Tensor _outer(const Tensor& input_a, const Tensor& input_b, const std::optional<
             a_slim,
             b_slim,
             std::nullopt,
-            tt::operations::primary::Matmul{
+            ttnn::operations::matmul::Matmul{
```

Comment:
You should be using `ttnn::matmul` here

Reply:
Ok. Done.
I also removed the nullopts/struct since the default parameters are nullopts